tools/tpp-run/tpp-run.cpp

```diff
@@ -147,6 +147,10 @@ llvm::cl::opt<bool>
                llvm::cl::desc("Kernel buffers are allocated on GPU"),
                llvm::cl::init(true));
 
+llvm::cl::opt<bool> defVectorToXSMM("vector-to-xsmm",
```

Comment:
Please do not put this here. This is the wrong place. None of the other options are here because this is not a tool option, but a pipeline option. All of the other pipeline options are in the pipeline code. You have already mentioned `lhsTile` and `rhsTile`, and they are not here either.

Reply:
lhsTile and rhsTile are explicitly added as pipeline options and are not bundles, as you pointed out yourself. gpubackend is a bundle and is added this way, and the way to add bundles is this. The only other way to add options to DefaultPipeline is like lhsTile, which is with a different name, which does not match vector-to-xsmm.